roles/common/templates/apt.list.j2

```diff
@@ -1,8 +1,15 @@
 # {{ansible_managed}}
 {% if node.platform is defined %}
+
+{% if node.platform == 'c8' %}{# ALT Linux 8 SP distribution repos. #}
+rpm [cert8] http://ftp.altlinux.org/pub/distributions/ALTLinux {{ node.platform }}/branch/{{ node.arch }} classic
```

Comment:
better to use repository base URL from `env.apt.sources` same as for other platforms